sysutils/libvirt/Portfile

```diff
@@ -75,11 +104,35 @@ variant avahi description {Use Avahi to advertise remote daemon} {
     configure.args-replace  --without-avahi --with-avahi
 }
 
+variant fuse description {FUSE support} {
+    depends_lib-append      port:osxfuse
+    configure.args-replace  --without-fuse --with-fuse
+}
+
+variant openssl description {Use OpenSSL instead of GnuTLS} {
+    depends_lib-replace     port:gnutls path:lib/libssl.dylib:openssl
+    configure.args-replace  --with-gnutls --without-gnutls
+    configure.args-replace  --without-openssl --with-openssl
+}
```

Comment:
I would prefer to also add a gnutls variant, and for the openssl and gnutls variants to conflict.